routing/pathfind_test.go

```diff
@@ -2306,6 +2309,17 @@ func TestRestrictOutgoingChannel(t *testing.T) {
 		t.Fatalf("expected route to pass through channel 2, "+
 			"but channel %v was selected instead", route.Hops[0].ChannelID)
 	}
+
+	// If a direct channel to target is allowed as well, that channel is
+	// expected to be selected because the routing fees are zero.
+	ctx.restrictParams.OutgoingChannelIDs = []uint64{2, 6}
```

Comment:
Where do these chan IDs come from? The order they're added to the graph above? If so would be nice to move them into a constant, or perhaps add a method to `[]*testChannel` which lets you look them up easily.

Reply:
constants added